README.md

```diff
@@ -1,3 +1,157 @@
 # redux-entitize
 
-Automated [redux](https://github.com/reactjs/redux)-state management for all your entities.
\ No newline at end of file
+Automated [redux](https://github.com/reactjs/redux)-state management for all your entities.
+
+`redux-entitize` maintains normalized entities in the state and provides:
+
+- an entities reducer
+- entity update / delete action creators
+- entity selectors
+
+## Getting started
+
+### Install
+
+```bash
+npm install --save redux-entitize
+```
+
+or
+
+```bash
+yarn add redux-entitize
+```
+
+### Usage Example
+
+`redux-entitize` provides a creator function for a single entities reducer handles the update and delete operations on your entities.
+Here is how you set up a reducer according to your `normalizr` schemas.
+
+```javascript
+// schemas.js
+
+import { normalize, schema } from 'normalizr';
+
+const userSchema = new schema.Entity('users');
+const commentSchema = new schema.Entity('comments', {
+  commenter: userSchema
+});
+
+export const schemas = {
+  "users": userSchema,
+  "comments": commentSchema,
+};
+```
+
+```javascript
+// reducers.js
+
+import { createEntitiesReducer } from 'redux-entitize';
+import { combineReducers } from 'redux';
+import { schemas } from 'schemas.js';
+
+const entitiesReducer = createEntitiesReducer(schemas);
+
+export const reducers = combineReducers({
+  someReducer: (state, action) => 42,
+  entitiesReducer
```

Comment:
1. `-Reducer`-suffix is a naming anti-pattern
2. redux-entitize **requires** the name of the state slice to be called `entities` for the selectors to work